lib/src/components/Banner.tsx

```diff
@@ -9,13 +9,71 @@ type TBanner = {
 	image?: string,
 	primaryButton?: ReactElement,
 	secondaryButton?: ReactElement,
-	children: ReactElement | ReactElement[],
+	children?: ReactElement | ReactElement[],
 	canClose?: boolean,
 	onClose?: () => void,
 	onClick?: () => void,
-	maxHeight?: string,
 	variant?: 'default' | 'image' | 'split' | 'background'
 }
+type TDefaultVariant = {
+	title?: string,
+	primaryButton?: ReactElement,
+	secondaryButton?: ReactElement,
+	children?: ReactElement | ReactElement[],
+	variant?: 'default' | 'image' | 'split' | 'background'
+}
+
+function	SplitVariant({image}: {image: string}) {
```

Comment:
@MarcoWorms I splitted the variants in subComponents to make it easier to read.

This one is for the split variant and is used **_ONLY_** for the image part (right part) to make it fit correctly